pkgs/development/ocaml-modules/magic-mime/default.nix

```diff
@@ -1,27 +1,23 @@
-{ stdenv, lib, fetchzip, ocaml, findlib, ocamlbuild }:
+{ lib, buildDunePackage, fetchFromGitHub }:
 
-let version = "1.0.0"; in
+buildDunePackage rec {
+  pname = "magic-mime";
+  version = "1.1.3";
 
-stdenv.mkDerivation {
-  pname = "ocaml-magic-mime";
-  inherit version;
-
-  src = fetchzip {
-    url = "https://github.com/mirage/ocaml-magic-mime/archive/v${version}.tar.gz";
-    sha256 = "058d83hmxd5mjccxdm3ydchmhk2lca5jdg82jg0klsigmf4ida6v";
+  src = fetchFromGitHub {
+    owner = "mirage";
+    repo = "ocaml-magic-mime";
+    rev = "v${version}";
+    sha256 = "sha256-izoiJ3wXTA4TgZR2jEwtDh3qkzE5vyoJqQx8HbtLdjI=";
   };
 
-  nativeBuildInputs = [ ocaml findlib ocamlbuild ];
-  buildInputs = [ findlib ];
-  configurePlatforms = [];
-
-  createFindlibDestdir = true;
+  minimumOCamlVersion = "4.03";
```

Comment:
Maybe you can use the brand new `minimalOCamlVersion`.

Reply:
Sure. From now on will use `minimalOCamlVersion`.